### ISSUE TITLE
Improvement | Grid Console to show total number of connected nodes

### ISSUE THREAD

**Issue by mstankovic86** [groupon/Selenium-Grid-Extras#206]
It would be great if we can see total number of connected nodes in Grid Console web page (https(s)://hub_ip:hub_port/grid/console)

**Comment by smccarthy** [comment]
@mstankovic86 Right now you can just do a search for "WebDriver" (or anything else that is the same for each node), right ?

**Comment by mstankovic86** [comment]
yes, i can, but there could be scenario where node is attached but it is not connected on the hub (picture bellow)

![screen shot 2016-05-03 at 10 48 27 am](https://cloud.githubusercontent.com/assets/8803779/14978607/4a6663e4-111d-11e6-9751-43433939219e.png)
it would be good if I can see total (reserved) number of nodes (browsers). For example:

nodes: 33
chrome: 99 (5)
firefox: 5 (1)
.
.
.

**Comment by smccarthy** [comment]
I like this idea.

**Comment by smccarthy** [comment]
Need to make sure we have an api available to get this information, to fulfill : https://github.com/groupon/Selenium-Grid-Extras/issues/200

**Comment by mitchloudenbeck** [comment]
There is an endpoint in the stock selenium hub that can give you some of this info. Do a get to http://yourHub:4444/grid/api/hub and you should see the free/total slotCounts (look at newSessionRequestCount to see queued tests). It doesn't have the granularity to show different browsers but you can get overview to determine if your grid is saturated.
